components/KiboFulfillmentOptions.vue

```diff
@@ -68,15 +62,17 @@ export default defineComponent({
   setup(_, context) {
     const nuxt = useNuxtApp()
     const pickupInStore = nuxt.nuxt2Context.$config.fullfillmentOptions.find(
-      (option) => option.value === "InStorePickup"
+      (option) => option.shortName === "Pickup"
     ).label
 
-    const selectFulfillment = (fulfillmentOption: Fulfillment) => {
-      context.emit("change", fulfillmentOption.value)
+    const selectFulfillment = (fulfillmentOption: FulFillmentOption | Fulfillment) => {
+      const shouldOpenModal = fulfillmentOption.shortName === "Pickup" && !fulfillmentOption.details
+      context.emit("radioChange", fulfillmentOption.shortName, shouldOpenModal)
     }
     const handleStoreLocatorClick = () => {
-      context.emit("click")
+      context.emit("changeStore", "Pickup", true)
```

Comment:
Instead of hard coding Pickup every where can we make enum or constant and use it?